language/Types.md

```diff
@@ -74,7 +74,16 @@ instance showPercentage :: Show Percentage where
 
 ## Functions
 
-Functions in PureScript are like their Javascript counterparts, but always have exactly one argument.
+Functions in PureScript are like their Javascript counterparts, but always have exactly one argument. Function declarations and lambdas (anonymous functions) with multiple arguments are shorthand for curried lambda abstractions. For example, the following are all equivalent:
+
+```purescript
+add x y = x + y
+add = \x y -> x + y
+add = \x -> \y -> x + y
+add = (+)
+```
+
+Despite this overwhelming convention, uncurried JavaScript functions are can be used through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library. For more information see [The Foreign Function Interface](http://www.purescript.org/learn/ffi/).
```

Comment:
Well … I think the statement needs *some* explanation for those who do not have functional programming backgrounds (the heresy! 😉 ). So we at least need to explain like, No guys, you really can give multiple arguments to functions!! It just works a bit differently. And at the type level we pretend like everything takes one argument. Maybe a link to a good explanation of currying would be appropriate?

I added the link to the library to encourage readers' natural curiosity. I agree, this file's job is not to explain everything, but I do think the different bits of documentation need to be better linked together to encourage exploration beyond "functions take a single argument". Because what are these types but functions that take more than one argument? Plus, even though they're supported by the compiler, it's all basically just JS/PS code running it! I actually think this library is a prime example of PureScript's flexibility and it would be awesome to encourage learners to be curious about all of this.

Thanks for the feedback, I will keep pondering it.